sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Azure.Monitor.OpenTelemetry.Exporter.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="System.Text.Json" />
     <PackageReference Include="OpenTelemetry" />
     <PackageReference Include="OpenTelemetry.Extensions.PersistentStorage" />
+    <PackageReference Include="OpenTelemetry.Extensions.AzureMonitor" />
```

Comment:
oops! do we need this! why would exporter depend on a Sampler !

Reply:
It's only a part of tests now.